core/tests/test_list.c

```diff
@@ -72,6 +73,20 @@ TEST_LIST = {
     // config
     {"test_upper_power_of_two", test_upper_power_of_two},
 
+    // doubly_linked_list_uint8
+    {"test_doubly_linked_list_uint8_new", test_doubly_linked_list_uint8_new},
```

Comment:
Should be `{"doubly_linked_list_uint8_new", test_doubly_linked_list_uint8_new},` since it is running tests we don't need to specify it in the print